modules/clightning-plugins/trustedcoin.nix

```diff
@@ -0,0 +1,26 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+let cfg = config.services.clightning.plugins.trustedcoin; in
+{
+  options.services.clightning.plugins.trustedcoin = {
+    enable = mkEnableOption "Trustedcoin (clightning plugin)";
+    package = mkOption {
+      type = types.package;
+      default = config.nix-bitcoin.pkgs.trustedcoin;
+      defaultText = "config.nix-bitcoin.pkgs.trustedcoin";
+      description = mdDoc ''
+        The package providing trustedcoin binaries. The trustedcoin will try to
```

Comment:
"The trustedcoin" sounds a bit odd

Reply:
I'd just write "`trustedcoin` will try to …", but also not a native English speaker, so take with a grain of salt :)